portmidi.go

```diff
@@ -88,5 +88,8 @@ func Time() Timestamp {
 
 // convertToError converts a portmidi error code to a Go error.
 func convertToError(code C.PmError) error {
```

Comment:
Convenience I guess? Some calls return a positive value as a result, and a
PmError (negative) otherwise. In that case it's worth checking and calling
this conditionally since you would otherwise return the value. Other
functions just return a PmError so it's easier to just wrap the call in
this. Could have a check at each call but it's more code overall.
On Thu, Mar 31, 2016 at 18:44 JBD <notifications@github.com> wrote:

> In portmidi.go
> <https://github.com/rakyll/portmidi/pull/26#discussion_r58152843>:
>
> > @@ -88,5 +88,8 @@ func Time() Timestamp {
> >
> >  // convertToError converts a portmidi error code to a Go error.
> >  func convertToError(code C.PmError) error {
>
> Shouldn't we handle this case at the call sites? Why do we call
> convertToError if code is non-negative?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rakyll/portmidi/pull/26/files/57a873c7bb920d3f3899cf573e687f12ba27583e#r58152843>
>